docs/source/basics/customize_docs.md

```diff
@@ -0,0 +1,68 @@
+# Adding Documentation to your Workflow
+
+While most of the metadata of a workflow will be encapsulated in a LatchMetadata object, we still require a docstring in the body of the workflow function which specifies both a short and long-form description.
+
+## One Line Description
+
+The first line of the workflow function docstring will get rendered in the sidebar of the workflow and the workflow explore tab as a brief description of your workflow's functionality. Think of this as summarizing the entirety of your workflow's significance into a single line.
+
+We recommend limiting your workflow description to one sentence, as longer descriptions are only partially rendered on the Workflows page.
+
+```python
+from latch import workflow
+
+@workflow
+def rnaseq(
+    ...
+):
+    """Perform alignment and quantification on Bulk RNA-Sequencing reads.
+
+    ...
+    """
+    ...
+```
+
+![Short Description](../assets/ui/one-line-description.png)
+
+## Long Form Description
+
+The body of the workflow function docstring is where you write long-form markdown documentation. This markdown will get rendered in the dedicated workflow "About" tab on your interface. Feel free to include links, lists, code blocks, and more.
```

Comment:
explain that this is full GitHub-flavored markdown support, link to https://github.github.com/gfm/

we also support KaTeX, subset of LaTeX for mathematical typesetting https://katex.org/docs/supported.html